File: Person.java
public class Person {
    String name;
    int age;

    // constructor method
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }
    // getter and setter methods
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public int getAge() {
        return age;
    }
    public void setAge(int age) {
        this.age = age;
    }

    public void message() {
        System.out.println("Hello, my name is " + name + " and I am " + age + " years old.");
    }
    // main method
    public static void main(String[] args) {
        // instance of Person class
        Person person = new Person("John", 25);
        // calling the message method
        person.message();
    }
}

File: Person.rb
class Person
    def initialize(name, age)
        @name = name
        @age = age
    end
    def name
        @name
    end
    def age
        @age
    end
    def name=(name)
        @name = name
    end
    def age=(age)
        @age = age
    end

    def message
        puts "#{@name} is #{@age} years old."
    end
end
person = Person.new("John", 25)
person.message
person.name = "Jane"
person.age = 26
person.message